packages/worker/src/worker-options.ts

```diff
@@ -362,6 +362,20 @@ export interface WorkerOptions {
    */
   maxConcurrentActivityTaskPolls?: number;
 
+  /**
+   * Specify the behavior of workflow task polling.
+   *
+   * @default A fixed maximum whose value is min(10, maxConcurrentWorkflowTaskExecutions).
+   */
+  workflowTaskPollerBehavior?: PollerBehavior;
```

Comment:
Experimental?

Reply:
Will leave these unmarked since I think these fields themselves are very unlikely to change, just the details of the autoscale behavior.